workers/deployment_manager/DeploymentManager/ConnectionService/Connector.cs

```diff
@@ -0,0 +1,119 @@
+using Improbable.Worker;
+using Improbable.Worker.Alpha;
+using System.Collections.Generic;
+
+namespace DeploymentManager
+{
+    public class Connector
+    {
+        public static string LocatorHost = "locator.improbable.io";
+        public static ushort LocatorPort = 444;
+
+        public Connection Connection { get; private set; }
+
+        private readonly string workerType;
+        private readonly Connection metaConnection;
+        private readonly string deploymentName;
+
+        public Connector(string workerType, string deploymentName, Connection metaConnection = null)
+        {
+            this.metaConnection = metaConnection;
+            this.workerType = workerType;
+            this.deploymentName = deploymentName;
+        }
+
+        public string ReceivePlayerIdentityToken(string developmentAuthToken)
+        {
+            var playerIdentityTokenResponse = DevelopmentAuthentication.CreateDevelopmentPlayerIdentityTokenAsync(
+            LocatorHost,
+            LocatorPort,
+            new PlayerIdentityTokenRequest
+            {
+                DevelopmentAuthenticationTokenId = developmentAuthToken,
+                DisplayName = workerType,
+                PlayerId = workerType,
+                UseInsecureConnection = false,
+            }).Get();
+
+
+            if (playerIdentityTokenResponse.Status != ConnectionStatusCode.Success)
+            {
+                Log.Print(LogLevel.Fatal, playerIdentityTokenResponse.Error, metaConnection);
```

Comment:
What is this `Log`?

Reply:
Also does a fatal log automatically then kill the program?

Reply:
And we can recover from some of the status codes I think?

Reply:
it's the basic log class that Jared made.